Makefile

```diff
@@ -1,6 +1,11 @@
 NAME    := argobots
 SRC_EXT := gz
 SOURCE  = https://github.com/pmodels/$(NAME)/releases/download/v$(VERSION)/$(NAME)-$(VERSION).tar.$(SRC_EXT)
-PATCHES  = $(NAME)-9d48af0840.patch $(NAME)-9d48af0840...89507c1f8c.patch
-
+ID_LIKE := $(shell . /etc/os-release; echo $$ID_LIKE)
```

Comment:
You don't need to assign this here because you are using `=` for `PATCHES` and not `:=` so `$(ID_LIKE)` will be defined when `$(PATCHES)` is resolved.